src/main/scala/org/allenai/plugins/DeployPlugin.scala

```diff
@@ -737,9 +737,9 @@ object DeployPlugin extends AutoPlugin {
       rootDeployDirectory, "-maxdepth", "1",
       // Set `find` to only report directory names.
       "-type", "d",
-      // Set `find` to only report directories named after this project (to avoid stopping other
-      // deployed projects).
-      "-name", s""""$projectName*"""",
+      // Set `find` to only report directories named after this project, with an optional numeric
+      // suffix (to avoid stopping other deployed projects).
+      "-regex", s""""./$projectName\(\-[0-9]+\)?"""",
```

Comment:
Context: the deploy plugin supports deploying multiple instances of a single project to a single host, and will place the copies in directories named `$projectName-1`, `$projectName-2`, ... etc. *unless* there's only one copy of the project, in which case the directory is just named `$projectName`.

With this functionality, it's possible that any number of old project instances could be running on a host when we deploy to it, so we need to stop all of those instances before deploying new code.  This `find` command does that, but it was also finding and stopping any other projects with the prefix `$projectName-` (i.e. `controller-demo` for `projectName = controller`).